packages/models-library/src/models_library/projects.py

```diff
@@ -0,0 +1,193 @@
+import sys
+from enum import Enum
+from pathlib import Path
+from typing import Dict, List, Optional, Union
+
+from pydantic import UUID4, BaseModel, EmailStr, Extra, Field, HttpUrl, constr
+
+from .services import KEY_RE, PROPERTY_KEY_RE, VERSION_RE
+
+current_file = Path(sys.argv[0] if __name__ == "__main__" else __file__).resolve()
+
+DATE_RE = r"\d{4}-(12|11|10|0?[1-9])-(31|30|[0-2]?\d)T(2[0-3]|1\d|0?[0-9])(:(\d|[0-5]\d)){2}(\.\d{3})?Z"
+
+
+class Connection(BaseModel):
+    nodeUuid: Optional[str]
+    output: Optional[str]
+
+
+class FilePickerOutput(BaseModel):
+    store: Union[str, int]  # simcore/datcore
+    dataset: Optional[str]
+    path: str
+    label: str  # name of the file
+
+
+class AccessEnum(str, Enum):
+    ReadAndWrite = "ReadAndWrite"

```

Comment:
NOTE: 

I wonder if we could use snake case in the python classes and set an option to use aliases when generating the schemas as I already did [here](https://github.com/ITISFoundation/osparc-simcore/blob/567ea3ec0b1cbdd82a79c9076e0717dc96b6afaf/services/web/server/src/simcore_service_webserver/settings.py#L70)

Not sure it would have a huge computational overload but would be more consistent with our design

Reply:
actually I don't even know what this AccessEnum is for??? is it used?

Reply:
Or, being an enum the following might be reasonable. 
```python
class AccessEnum(str, Enum):
    READ_AND_WRIE = "ReadAndWrite"
```

But I understand why you would keep it per currently in the source.

Reply:
well guys. this is all very nice. but this PR just moved these sources around. I fixed what I could but this one will be done at a later point probably when we fix the inputs/outputs ports access rights.

Reply:
ok